<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-8</y>
      <width>801</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_11">
     <widget class="QWidget" name="detalhesWidget" native="true">
      <property name="geometry">
       <rect>
        <x>279</x>
        <y>9</y>
        <width>521</width>
        <height>551</height>
       </rect>
      </property>
      <widget class="QLabel" name="lblDetalheFotoDisplay">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>209</y>
         <width>421</width>
         <height>281</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Sem Imagem</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>510</y>
         <width>195</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btnFotoAnterior">
          <property name="text">
           <string>&lt; Anterior</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnFotoProxima">
          <property name="text">
           <string>Próxima &gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>3</y>
         <width>521</width>
         <height>207</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="lblDetalheNome">
              <property name="font">
               <font>
                <pointsize>16</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Selecione uma Viagem</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblDetalheDestino">
              <property name="text">
               <string>Destino:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblDetalheData">
              <property name="text">
               <string>Data:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="lblDetalheData_2">
              <property name="text">
               <string>Descrição:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextBrowser" name="browserDetalheDescricao"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="lblDetalheData_3">
          <property name="text">
           <string>Fotos:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QListWidget" name="listWidgetViagens">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>261</width>
        <height>551</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_12">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>220</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TESTE</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>812</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuViagens">
    <property name="title">
     <string>&amp;Viagens</string>
    </property>
    <addaction name="actionNova_Viagem"/>
   </widget>
   <widget class="QMenu" name="menuSobre">
    <property name="title">
     <string>Sobre</string>
    </property>
   </widget>
   <addaction name="menuViagens"/>
   <addaction name="menuSobre"/>
  </widget>
  <action name="actionNova_Viagem">
   <property name="text">
    <string>&amp;Criar Nova Viagem</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
